src/components/MyAccount/profileTabs.test.tsx

```diff
@@ -61,7 +61,7 @@ describe("ProfileTabs", () => {
         activePath="checkouts"
       />
     )
-    fireEvent.click(getByText("Requests"))
+    fireEvent.click(getByText("Requests (2)"))
```

Comment:
This is fine as is, but for the future, you can pass in a second options param with `{exact: false}` if it's something like this that is liable to change if we have more test cases.

Reply:
Yes, another option is `getByText(/requests/i)`